### PR TITLE
Added new Xiaomi Air Quality Monitor

### PR DESCRIPTION
**Pull request by fwestenberg**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27735

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by syssi — APPROVED** [review]
LGTM!

**Comment by klaasnicolaas** [comment]
We need to revert or change this!

**Comment by frenck** [comment]
@syssi This should not have been merged in this state, there are multiple things incorrect here.

File: source/_integrations/air_quality.xiaomi_miio.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Please follow the instructions on [Retrieving the Access Token](/integrations/vacuum.xiaomi_miio/#retrieving-the-access-token) to get the API token.
+
+## Configuration
+
+To add a Xiaomi Mi Air Quality Monitor to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+air_quality:
```
Comment (klaasnicolaas) [inline source/_integrations/air_quality.xiaomi_miio.markdown:33]: Only required config variables in the first example

File: source/_integrations/air_quality.xiaomi_miio.markdown [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  - platform: xiaomi_miio
+    name: Xiaomi Air Quality Monitor
+    host: 192.168.130.73
```
Comment (klaasnicolaas) [inline source/_integrations/air_quality.xiaomi_miio.markdown:36]: Host: IP_ADDRESS

File: source/_integrations/air_quality.xiaomi_miio.markdown [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Xiaomi Air Quality Monitor"
+description: "Instructions how to integrate your Xiaomi Mi Air Quality Monitor within Home Assistant."
+logo: xiaomi.png
+ha_category:
+  - Health
+ha_iot_class: Local Polling
+ha_release: 0.100.4
```
Comment (klaasnicolaas) [inline source/_integrations/air_quality.xiaomi_miio.markdown:8]: Wrong release

**Comment by frenck** [comment]
We don't accept platform-specific docs anymore, please merge all a single integration page.

**Pull request merged**